index.css

```diff
@@ -0,0 +1,100 @@
+*{
+    margin: 0;
+    padding: 0;
+    box-sizing: border-box;
+}
+
+body{
+    background-color: rgb(238, 238, 238);
+}
+
+.navbar{
+    padding: 10px 50px;
+    margin-top: 15px;
+}
+img{
+    width: 100px;
+    height: 50px;
+}
+
+.nav-items a{
+    margin-left: 50px;
+    font-size: 22px;
+    color: rgb(43, 42, 42);
+    
+}
+
+a{
+    text-decoration: none;
```

Comment:
OPTIONAL :
Work on the responsiveness, so it can display well on other screen sizes.